crates/biome_analyze/src/suppressions.rs

```diff
@@ -333,6 +338,22 @@ impl RangeSuppressions {
         false
     }
 
+    /// Checks if there's suppression that suppresses the current plugin in the range provided
```

Comment:
I'd reword this comment, given that the method does more than just checking for suppressions.

```suggestion
    /// Suppresses the plugin with the given `plugin_name` if there is a suppression comment
    /// for the given `position`.
    ///
    /// Returns `true` if a matching suppression comment was found, `false` otherwise.
```